frontend/apps/reader/modules/readerdictionary.lua

```diff
@@ -64,20 +65,90 @@ local function tidy_markup(results)
     return results
 end
 
+function ReaderDictionary:cleanSelection(text, is_single_word)
```

Comment:
I just made a mistake in ReaderDictionary:stardictLookup() where it should be called with `false`. I assume stardict needs a word, without spaces, unlike wikipedia where multiple words are ok. This `is_single_word` is there so we can do different cleaning for each cases.